packages/api/src/base/Init.ts

```diff
@@ -200,6 +200,7 @@ export abstract class Init<ApiType extends ApiTypes> extends Decorate<ApiType> {
     );
 
     if (header.parentHash.isEmpty) {
+      l.warn(`Unable to retrieve header ${blockHash.toString()} and parent ${header.parentHash.toString()} from supplied hash`);
```

Comment:
```suggestion
      l.debug(`Unable to retrieve header ${blockHash.toString()} and parent ${header.parentHash.toString()} from supplied hash`);
```

The if you do `export DEBUG=api/init` it will show up.